specifyweb/frontend/js_src/lib/components/FormEditor/spec.ts

```diff
@@ -64,8 +146,11 @@ const resolvedViewSpec = () =>
         businessRules: localized(
           typeof table === 'object'
             ? `edu.ku.brc.specify.datamodel.busrules.${
-                businessRules[table.name] ?? table.name
-              }BusRules`
+                businessRules[table.name] ||
```

Comment:
@combs-a is correct in https://github.com/specify/specify7/pull/4816#pullrequestreview-2018353684
if current table does not have business rules, you should return undefined here, rather than a broken string